### PR TITLE
Fix documented type of mode field in copy module

### PR DESCRIPTION
**Pull request by MSDehghan**
##### SUMMARY
Type of 'mode' field in Copy module is 'path' which is absolutely wrong. I changed it to 'str'.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
copy

### CODE REVIEW

File: lib/ansible/modules/copy.py [hunk 1]
```diff
@@ -75,7 +75,7 @@
     - As of Ansible 2.3, the mode may also be the special string C(preserve).
     - C(preserve) means that the file will be given the same permissions as the source file.
     - When doing a recursive copy, see also C(directory_mode).
-    type: path
+    type: str
```
Comment (sivel) [inline lib/ansible/modules/copy.py]: Mode is actually `type='raw'`

https://github.com/ansible/ansible/blob/f367cba1ab202ca43ee0b140e042dc00a41c144d/lib/ansible/module_utils/basic.py#L244

```suggestion
    type: raw
```

**Comment by Shrews** [comment]
After discussing this in triage today, we suggest removing `type` altogether, which will pick of the doc fragment value. Overriding `description` is sufficient.

**Comment by MSDehghan** [comment]
Sorry. I didn't get it.
You mean removing `type` for all options in copy module? And Why description is enough? Where does it come from then?

**Comment by s-hertel** [comment]
@MSDehghan It comes from the doc fragment here https://github.com/ansible/ansible/blob/stable-2.10/lib/ansible/plugins/doc_fragments/files.py#L17. The description for `mode` has some additional info for this module, so overwriting that field is helpful. If you omit the `type` field here, the value in the doc fragment will be used. You can test this out with `ansible-doc copy`.

**Comment by MSDehghan** [comment]
@s-hertel Thank you for the explanation. I removed it.

**Comment by s-hertel** [comment]
Thanks for the fix!

**Pull request merged**